### PR TITLE
Bug 1173268 - [Contacts][NGA] Move 'updatePhoto' out of contacts.js

### PR DESCRIPTION
**Pull request by fcampo**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=88dc3642fc7e82f452bed98f5da24cb3bdfe6548

File: apps/communications/contacts/js/views/details.js [hunk 1]
```diff
@@ -609,14 +609,14 @@ contacts.Details = (function() {
     if (photo) {
       var currentHash = cover.dataset.imgHash;
       if (!currentHash) {
-        Contacts.updatePhoto(photo, cover);
+        utils.photo.update(photo, cover);
```
Comment (borjasalguero) [inline apps/communications/contacts/js/views/details.js]: I would change this to `utils.dom.setBackground(photo, element)`

File: apps/communications/contacts/js/utilities/photo.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+(function(exports) {
+
+  exports = exports || {};
+
+  var PhotoUtils = {
+    update: function updatePhoto(image, dest) {
+      console.log('>> UPDATING PHOTO > ' + image + '-' + dest);
```
Comment (borjasalguero) [inline apps/communications/contacts/js/utilities/photo.js]: Remove this log :)

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6f7dc6f76ed0e8a3e28e7cceb298e5c02d8fb052

**Pull request merged**